### PR TITLE
Some React Configuration Changes

### PR DESCRIPTION
**Pull request by laura-king**
- remove the hashes from environment.yml
- change react port to 8155 (used on dev server on vm, set by murali)
- change various configuration for next js
- add link wrapper to accommodate url base path differences in different versions
- update readme

### CODE REVIEW

**Comment by tangkong** [comment]
This looks mostly good to me, I'll wait until this is up on the dev server to give it a spin

**Comment by tangkong** [comment]
My clicking around went smoothly, and this mostly works as deployed.  Requesting a review from @GreatDanton for a quick check of code quality

File: react/.env.development [hunk 1]
```diff
@@ -5,4 +5,6 @@
 # with 'NEXT_PUBLIC_'
 
 # url of the archive server with a REST API
+
+NEXT_PUBLIC_BASE_PATH=''
```
Comment (GreatDanton) [inline react/.env.development:9]: nitpick: I would display these fields in the same order for both .env files

**Comment by GreatDanton** [comment]
Looks fine to me.

**Pull request merged**